packages/manager/cypress/e2e/core/cloudpulse/cloudpulse-navigation.spec.ts

```diff
@@ -55,7 +55,7 @@ describe('CloudPulse  navigation', () => {
   /*
    * - Confirms that manual navigation to Cloudpulse landing page with feature is disabled displays Not Found to user.
    */
-  it('displays Not Found when manually navigating to /cloudpulse with feature flag disabled', () => {
+  it.skip('displays Not Found when manually navigating to /cloudpulse with feature flag disabled', () => {
```

Comment:
Same here, make the PR code review ready, is this ready for review or not working?

Reply:
Skipped test cases related to ACLP flag validation as this validation has been removed in the integration branch. These test cases were failing and also taking a considerable amount of time to execute. Skipping them will help streamline the testing process